macro.pfsi/src/Event-Move.cpp

```diff
@@ -69,6 +72,27 @@ e_move_returnHome::e_move_returnHome(const double tEvent_, human* h) :
     double trip_t = tEvent_ + R::rexp(1./h->get_trip_frequency());
     h->addEvent2Q(e_move_takeTrip(trip_t,dest_id,h));
 
+    /* vaccinate/treat upon travel */
+    if(h->get_tile()->get_patch(dest_id)->get_reservoir()){
+
+      /* vaccine */
+      if(h->get_tile()->get_params()->get_travel_vaxx()){
+
+        /* prophylaxis? */
+        bool treat(h->get_tile()->get_params()->get_travel_treat());
+        h->addEvent2Q(e_pfsi_pevaxx(trip_t - 1E-10, treat, h));
```

Comment:
So if I understand this, in the event that we travel, we check whether or not travel_vaxx and travel_treat are on, and then we schedule those events immediately before the trips are taken? Ie, as someone gets on a plane to go to the mainland, we treat them?

As I'm typing this, it looks like I forgot to communicate one thing: we only vaccinate people who go to the mainland! We'll need to check that they're going to the right destination.

Reply:
At the moment we schedule our next trip away from home, we check if the destination is a reservoir patch. If and only if it's a reservoir, do we vaccinate or treat (or do neither, if both parameters are set to FALSE) that person.

Reply:
The t-10^-10 isn't pretty, but it works. BTW, there are pi x 10^7 seconds in a year, so this is about 3 milliseconds before leaving.

Reply:
Sounds good